backend/app/api/v1/config.py

```
@@ -136,3 +136,34 @@ def list_configs(config_definition_key: str, page: int = 1, limit: int = 10):
+    dict
+        A success message.
+    """
+    try:
+        u_config(
+            config_definition_key,
+            config_key,
+            config.data,
+        )
+    except ValueError as e:
+        logger.warning(f"Configuration not found: {e}")
+        raise HTTPException(status_code=404, detail=str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)